schema/home/home_page_artist_module.js

```diff
@@ -7,40 +7,48 @@ import gravity from '../../lib/loaders/gravity';
 import { NodeInterface } from '../object_identification';
 import { toGlobalId } from 'graphql-relay';
 import {
+  GraphQLEnumType,
   GraphQLID,
   GraphQLNonNull,
   GraphQLList,
   GraphQLObjectType,
   GraphQLString,
 } from 'graphql';
 
-export const Results = {
-  suggested: {
+// This object is used for both the `key` argument enum and to do fetching.
+export const HomePageArtistModuleTypes = {
+  SUGGESTED: {
+    description: 'Artists recommended for the specific user.',
     fetch: (accessToken, userID) => {
       return gravity.with(accessToken)(`user/${userID}/suggested/similar/artists`);
     },
     display: (accessToken, userID) => {
+      const fetch = HomePageArtistModuleTypes.SUGGESTED.fetch;
       return Promise.resolve(!!(accessToken && userID)).then(display => {
-        return display && Results.suggested.fetch(accessToken, userID).then(results => {
+        return display && fetch(accessToken, userID).then(results => {
           return results.length > 0;
```

Comment:
Come to think of it, maybe this is a little bit easier to parse like
```
return fetch(accessToken, userID).then(results => {
   return results.length > 0 && display;
});
```

I'm not quite sure what passing a Boolean and a Promise will do to a resolve, either way (if it works or not) it's weird to think about :)